cfpb/consumerfinance.gov
# Notification: add type option and make available on Browse & Learn pages

Adds the notification module to Browse & Learn page types so it can be used on coronavirus pages. On these page types, we think notifications will be used towards the top of the page (like the FCM), so the module is included in the Header rather than Content section. 

This PR also reintroduces the notification type dropdown that was removed in #4977 since there's interest in using the default (gray) style on coronavirus pages to highlight information in a way that's more attention-grabbing than a well but less alarming than the warning style. The type dropdown only includes the default (gray) and warning styles -- there isn't a clear need for the success or error style -- and the warning style continues to be the default since the fact that it was the only option for 1.5+ years suggests it's likely to be the most common use case. 

**Note:** The default gray notification style is called `Information` in the dropdown to avoid confusion since it's the default style for our notifications but not the default option of the notification Wagtail block.


## Additions

- Notification module on Browse and Learn page types
- Type dropdown in notification module


## How to test this PR

1. Check out branch and run migrations
2. Create a new Browse or Learn page, add a notification, and verify that default `Information` style can be applied to it


## Screenshots
### Past notification on coronavirus page:
<img width="724" alt="Screen Shot 2021-03-12 at 12 18 41 PM" src="https://user-images.githubusercontent.com/778171/110994552-c7930480-832d-11eb-9e37-e53b14a3d00e.png">

### Future approach:
<img width="823" alt="Screen Shot 2021-03-12 at 12 25 46 PM" src="https://user-images.githubusercontent.com/778171/110994762-22c4f700-832e-11eb-9c47-0d157c332ce6.png">


## Notes and todos
-  Should add a test for the notification type


## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
